poc/vdaf_pine.py

```diff
@@ -299,7 +299,7 @@ def prep_init(Pine,
             Pine.joint_rand_part_and_seed_for_aggregator(
                 agg_id,
                 k_wr_joint_rand_blind,
-                meas_share[:Pine.Flp.Valid.encoded_gradient_len],
+                meas_share[:Pine.Flp.Valid.encoded_gradient_and_norm_len],
```

Comment:
Sorry. I didn't re-run tests because I was finishing the spec in an environment where I don't have Sage :)